src/wasm/wasm-debug.cpp

```diff
@@ -877,6 +894,7 @@ static void updateLoc(llvm::DWARFYAML::Data& yaml,
   BinaryLocation base = 0;
   for (size_t i = 0; i < yaml.Locs.size(); i++) {
     auto& loc = yaml.Locs[i];
+    base = locationUpdater.getLocationBaseAddress(loc.Offset);
```

Comment:
the `BinaryLocation ` could be here too? (we don't need anything outside of the loop?)

Reply:
done